src/_pytest/nodes.py

```diff
@@ -174,6 +174,11 @@ def ihook(self):
         """ fspath sensitive hook proxy used to call pytest hooks"""
         return self.session.gethookproxy(self.fspath)
 
+    @property
+    def own_markers(self) -> List[Mark]:
+        """The marker objects belonging to this node."""
```

Comment:
I realize the property name can't be changed but why "markers" rather than "marks"?

Reply:
another critical detail i just realized, changing own_markers from a real object to a transient temporal property is breaking api change that breaks users that change own_markers directly

Reply:
@blueyed

I would extend the comment here. Suggestion (but please verify what I write!):

```
The marks directly belonging to this node.

This includes marks manually added with `add_marker()`, and marks directly applied
to the object (using `@pytest.mark.foo` decorations for example). It does not include
marks inherited from parent nodes (for example, a mark applied to a class node is not
included in a method's `own_markers`).
```

Would also add a note about modifying it, depending on the answer to the next question.

BTW, is there actually any significance to the the order or marks?

@RonnyPfannschmidt

> changing own_markers from a real object to a transient temporal property is breaking api change

Do you think changing `own_markers` directly is something that should be supported, or deprecated?

Reply:
there is significance to the order of marks in the sense, that they apply in the order they are declared/applied

this is important for details like selection of correct messages/failures

in retrospect i believe exposing own_markers was a mistake, however right now its public and mutable